build_projects/dotnet-cli-build/MsiTargets.cs

```diff
@@ -258,5 +172,11 @@ private static void ReattachEngineToBundleHelper(string bundle, string engine)
 
             File.Delete(engine);
         }
+
+        private static string GetEngineName(string bundle)
```

Comment:
going to leave this as is, it is not breaking anything and reverting this file would require redoing the changes.

---
In reply to: [64635837](https://github.com/dotnet/cli/pull/3230#discussion_r64635837) [](ancestors = 64635837)